backend/src/scripts/feature-extractor-script.js

```diff
@@ -0,0 +1,14 @@
+const { parseProductsFromCsvPath } = require('../utilities');
+const { databaseConnectUsingEnv } = require('../database');
+require('dotenv').config();
+
+databaseConnectUsingEnv();
+Promise.all(parseProductsFromCsvPath('./resources/medshare-data-sample-reformat.csv'))
```

Comment:
Think we should drop all existing collections before doing all these inserts

Reply:
I was debating about that. What if the user wants to process another CSV?